src/editor/codemirror/lint/lint.ts

```diff
@@ -92,7 +95,7 @@ class SelectedDiagnostic {
   constructor(readonly from: number, readonly to: number, readonly diagnostic: Diagnostic) {}
 }
 
-class LintState {
+export class LintState {
```

Comment:
I don't think the `LintState` class is imported anywhere.

Reply:
We rely on the export of lintState so it makes sense to export the type it returns.